### DISCUSSION TITLE
UMbraco V15 looks like a very bad joke

### DISCUSSION THREAD

**Discussion by cmwalolo** [umbraco/Umbraco-CMS#17859] [Features and ideas]

**Comment by nul800sebastiaan** [comment]
Hey there @cmwalolo - we'd love to address any feedback you have, but this is not the way. 
I'd like to remind you of our [code of conduct](https://github.com/umbraco/Umbraco-CMS?tab=coc-ov-file#readme) and ask you to provide feedback based on those guidelines in the future. 
Thanks!
